mirgecom/prometheus.py

```diff
@@ -1,9 +1,9 @@
-import numpy as np
+import numpy as realnp
```

Comment:
For clarity I think `import numpy as np` should be used.  And then a different name for the pseudo numpy.